### PR TITLE
Tests running, adding 'creation_date' and 'shared_users', field. reformatting 'markdown_text' 

### PR DESCRIPTION
**Pull request by slebras**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@ docker==3.7.0
 elasticsearch==5.5.3
 PyYAML==5.1
 requests==2.21.0
+beautifulsoup4==4.6.3
```
Comment (jayrbolton) [inline requirements.txt]: why not the latest, 4.7.1?

File: requirements.txt [hunk 2]
```diff
@@ -5,3 +5,5 @@ docker==3.7.0
 elasticsearch==5.5.3
```
Comment (jayrbolton) [inline requirements.txt]: can we delete the elasticsearch dependency?

File: src/index_runner/indexers/indexer_utils.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+from kbase_workspace_client import WorkspaceClient
+from kbase_workspace_client.exceptions import WorkspaceResponseError
+
+from ..utils.config import get_config
+
+
+def get_shared_users(data):
+    """
+    get shared users
```
Comment (jayrbolton) [inline src/index_runner/indexers/indexer_utils.py]: make the docstring something like "Returns a list of users that have read or write access to the given workspace object."

File: src/test/test_integration.py [hunk 4]
```diff
@@ -131,16 +131,32 @@ def test_narrative_update_event(self):
         print('..finished producing, now consuming. This may take a couple minutes as the workers restart...')
         msg_data = consume_last(config['topics']['elasticsearch_updates'])
 
-        self.assertEqual(msg_data['doc'], {
+        print("="*80)
```
Comment (jayrbolton) [inline src/test/test_integration.py]: do you want to keep this comment wall

File: src/test/test_integration.py [hunk 5]
```diff
@@ -131,16 +131,32 @@ def test_narrative_update_event(self):
...
+        print("="*80)
+        print("="*80)
+        print("doc message: ", msg_data['doc'])
+        print("="*80)
+        print("="*80)
+        print("="*80)
+        # TODO: update missing fields "accgrp", "shared_users", "creation_date", and "public"
```
Comment (jayrbolton) [inline src/test/test_integration.py]: Can you change `"accgrp"` to `"access_group"`?

**Comment by jayrbolton** [comment]
I'm done commenting now

File: src/test/test_integration.py (hunk 4 above)
Comment (slebras) [reply src/test/test_integration.py]: no, i apologize

**Pull request closed without merging**